src/components/movie-viewer/MovieDetails.jsx

```diff
@@ -1,24 +1,29 @@
 import React, { useState, useEffect } from "react";
+import { collection, getDocs, getFirestore } from "firebase/firestore";
 import { useParams } from "react-router-dom";
-// import moviesData from "./movies_data.json";
+import firebase from "../../firebase-config";
 import "./style/MovieDetails.scss";
 
 export default function MovieDetails() {
   const [movie, setMovie] = useState([]);
 
   const { id } = useParams();
-  const apiKey = "46e8c41c";
-  const apiUrl = `http://www.omdbapi.com/?apikey=${apiKey}&i=${id}`;
 
-  const getData = () => {
-    fetch(apiUrl)
-      .then((response) => response.json())
-      .then((data) => setMovie(data));
+  const db = getFirestore(firebase);
+
+  const getData = async () => {
+    const querySnapshot = await getDocs(collection(db, "movies"));
```

Comment:
here you download all movies from firebase, and ten you select only one. You can get a single document from firebase:
https://firebase.google.com/docs/firestore/query-data/get-data
 ```
const docRef = doc(db, "movies", movieId);
const docSnap = await getDoc(docRef);
```

Reply:
Yes but id is now not part of the collection document, it is added to our object for component from seperate firebase object.

Reply:
WIdzialam to samo w dokumentacji i testowalam to ale to wyzej movieID moze sie odnosic tylko do obietu dokument ktory ma jedynie imdbID. 
Po pobraniu snap moge do obiektu danych dodac doc.id jak zrobilam w MovieBrowser
To co mi preszlales by dzialalo i dzialalo jak uzywalismy imdbID.

Reply:
ale przecież masz id w tym kompnencie bierzesz go z URL (np. http://localhost:1234/movie/0)
`  const { id } = useParams();`

Reply:
![WhatsApp Image 2022-02-09 at 19 15 06](https://user-images.githubusercontent.com/95184866/153264351-ff446693-0cf2-403e-8285-4a34faea69b9.jpeg)

tak ale nie mam go w danych - dokument w kolekcji ma id ale nie ma go w danych samego dokumentu ktore pobieraja sie uzywajac tej funkcji firebasa ktora wkleiles wyzej

Reply:
dalam ci screenshot: 
kolumna movies = doc.id 
ostatnia kolumna = doc.data